src/i18n/en-actions.ts

```diff
@@ -33,6 +33,7 @@ const actionsMessageDescriptors = {
       ${ColonyExtendedActions.AddressBookUpdated} {Address book was updated}
       ${ColonyExtendedActions.TokensUpdated} {Colony tokens were updated}
       ${ColonyExtendedActions.SafeRemoved} {Remove Safe}
+      ${ColonyExtendedActions.SafeAdded} {Add Gnosis Safe}
```

Comment:
We might have a different chain than Gnosis so either you remove Gnosis from the string or you add a variable and pass the relevant chain name.

Reply:
This would be: "Add Safe From Rinkeby Testnet"